wrappers/python/python.cpp

```diff
@@ -143,177 +143,175 @@ PYBIND11_MODULE(NAME, m) {
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_extrinsics, rotation, float, 9), "Column - major 3x3 rotation matrix")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_extrinsics, translation, float, 3), "Three-element translation vector, in meters")
         .def("__repr__", [](const rs2_extrinsics &e) {
-        std::stringstream ss;
-        ss << "rotation: " << array_to_string(e.rotation);
-        ss << "\ntranslation: " << array_to_string(e.translation);
-        return ss.str();
-    });
+            std::stringstream ss;
+            ss << "rotation: " << array_to_string(e.rotation);
+            ss << "\ntranslation: " << array_to_string(e.translation);
+            return ss.str();
+        });
 
-    py::class_<rs2_intrinsics> intrinsics(m, "intrinsics");
+    py::class_<rs2_intrinsics> intrinsics(m, "intrinsics", "Video stream intrinsics.");
     intrinsics.def(py::init<>())
-        .def_readwrite("width", &rs2_intrinsics::width)
-        .def_readwrite("height", &rs2_intrinsics::height)
-        .def_readwrite("ppx", &rs2_intrinsics::ppx)
-        .def_readwrite("ppy", &rs2_intrinsics::ppy)
-        .def_readwrite("fx", &rs2_intrinsics::fx)
-        .def_readwrite("fy", &rs2_intrinsics::fy)
-        .def_readwrite("model", &rs2_intrinsics::model)
-        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_intrinsics, coeffs, float, 5))
-        .def("__repr__", [](const rs2_intrinsics& self)
-    {
-        std::stringstream ss;
-        ss << "width: " << self.width << ", ";
-        ss << "height: " << self.height << ", ";
-        ss << "ppx: " << self.ppx << ", ";
-        ss << "ppy: " << self.ppy << ", ";
-        ss << "fx: " << self.fx << ", ";
-        ss << "fy: " << self.fy << ", ";
-        ss << "model: " << self.model << ", ";
-        ss << "coeffs: " << array_to_string(self.coeffs);
-        return ss.str();
-    });
-
-    py::class_<rs2_motion_device_intrinsic> motion_device_inrinsic(m, "motion_device_intrinsic");
+        .def_readwrite("width", &rs2_intrinsics::width, "Width of the image in pixels")
+        .def_readwrite("height", &rs2_intrinsics::height, "Height of the image in pixels")
+        .def_readwrite("ppx", &rs2_intrinsics::ppx, "Horizontal coordinate of the principal point of the image, as a pixel offset from the left edge")
+        .def_readwrite("ppy", &rs2_intrinsics::ppy, "Vertical coordinate of the principal point of the image, as a pixel offset from the top edge")
+        .def_readwrite("fx", &rs2_intrinsics::fx, "Focal length of the image plane, as a multiple of pixel width")
+        .def_readwrite("fy", &rs2_intrinsics::fy, "Focal length of the image plane, as a multiple of pixel height")
+        .def_readwrite("model", &rs2_intrinsics::model, "Distortion model of the image")
+        .def_property(BIND_RAW_ARRAY_PROPERTY(rs2_intrinsics, coeffs, float, 5), "Distortion coefficients, order: k1, k2, p1, p2, k3")
```

Comment:
Simply write "Distortion coefficients". What they represent depends on the distortion model